website/docs/r/container_cluster.html.markdown

```diff
@@ -362,6 +362,12 @@ The `node_config` block supports:
   * `monitoring` (`https://www.googleapis.com/auth/monitoring`),
     if `monitoring_service` points to Google
 
+  In addition to the `oauth_scopes`, the `service_account` which is used by the Node VMs
```

Comment:
Correct me if I'm wrong (I'm not too familiar with `oauth_scopes`), but this seems like it has more to do with `service_account`. If we move this block to service account, we could go from:

```markdown
 * `service_account` - (Optional) The service account to be used by the Node VMs.
     If not specified, the "default" service account is used.
```

```markdown
 * `service_account` - (Optional) The service account to be used by the Node VMs.
If not specified, the "default" service account is used.
The service account being used needs the [roles/logging.logWriter](https://cloud.google.com/iam/docs/understanding-roles#stackdriver_logging_roles)
and [roles/monitoring.metricWriter](https://cloud.google.com/iam/docs/understanding-roles#stackdriver_monitoring_roles)
roles.

-> Projects that enable the API with Terraform may need these roles added manually. Projects that enable the API in the Cloud Console should have them added automatically.
```

Reply:
Makes sense to move it there. However it should highlight, what these permissions are needed for. I will update the pull request accordingly.